src/view.cpp

```diff
@@ -354,6 +354,14 @@ nonstd::optional<std::uint8_t> View::open_in_browser(const std::string& url)
 	Stfl::reset();
 	const auto ret = utils::run_interactively(cmdline, "View::open_in_browser");
 	pop_current_formaction();
+	auto form = formaction_stack[current_formaction];
```

Comment:
The `SIGSEGV` crashes happened in the tests because there were no formactions put on the stack (so, even with `current_formaction = 0`, the index was invalid).

Fixed in the succeeding commits.